packages/spor-react/src/theme/utils/background-utils.ts

```diff
@@ -26,3 +20,31 @@ export function baseBackground(state: State, props: StyleFunctionProps) {
       return {};
   }
 }
+
+type GhostBackgroundState = Subset<State, "default" | "hover" | "focus" | "active" | "selected">;
+
+export function ghostBackground(state: GhostBackgroundState, props: StyleFunctionProps) {
+  switch (state) {
+    case "hover": {
+      return {
+        backgroundColor: mode("seaMist", "whiteAlpha.100")(props),
+      }
+    }
+    case "active":
+      return {
+        backgroundColor: mode("seaMist", "whiteAlpha.200")(props),
+      }
+    case "focus":
+      return {
+        backgroundColor: mode("transparent", "transparent")(props),
```

Comment:
Ser at det er digg å være konsekvent, men her kan vi kanskje forenkle til 

```suggestion
        backgroundColor: "transparent",
```